plugins/techdocs/src/alpha.tsx

```diff
@@ -31,7 +31,10 @@ import {
   fetchApiRef,
   identityApiRef,
 } from '@backstage/core-plugin-api';
-import { convertLegacyRouteRef } from '@backstage/core-compat-api';
+import {
+  compatWrapper,
+  convertLegacyRouteRef,
```

Comment:
Makes me think about the naming... `convertLegacyX` for some X might be nice? Or is is just too unclear what X should be then